### DISCUSSION TITLE
Student developer package

### DISCUSSION THREAD

**Discussion by TaniyaPuri** [community/community#109541] [GitHub Education] [answered]
### Hi everyone,

When will i get the access of student developer package

**Comment by namith1003** [comment, marked as answer]
It may take some time cause there have been students reporting times as short as 2 days, some stating 4 days and some not having any confirmation even after 8 days, you may have to be a bit patient or maybe contact the GitHub team to request in the scenario you feel that it has taken too long.

Hopefully this helps answer your question.

**Comment by johnenes** [comment]
how can get access to .tech on student developer package from the alx
